package.json

```diff
@@ -24,8 +24,8 @@
     "lint": "lerna run lint",
     "lint:eslint": "lerna run lint:eslint",
     "postpublish": "lerna run --stream --concurrency 1 postpublish",
-    "test": "lerna run --ignore @sentry-internal/* test",
-    "test-ci-browser": "lerna run test --ignore \"@sentry/{node,opentelemetry-node,serverless,nextjs,remix,gatsby}\" --ignore @sentry-internal/*",
+    "test": "lerna run test --ignore @sentry-internal/*",
+    "test-ci-browser": "yarn test --ignore \"@sentry/{node,opentelemetry-node,serverless,nextjs,remix,gatsby}\"",
```

Comment:
This was just a minor thing, I though it might be good for `test-ci-browser` to directly extend `test`